src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -401,6 +410,101 @@
     Collection<String> getDefaultGameRules();
 
     /**
+     * Gets the {@link Statistic} with the specified id.
+     *
+     * @param id The id of the statistic to return
+     * @return The statistic or Optional.absent() if not found
+     */
+    Optional<Statistic> getStatistic(String id);
+
+    /**
+     * Gets the {@link Statistic} for the given {@link StatisticGroup} and
+     * {@link EntityType}. If the statistic group is not a valid
+     * {@link EntityStatistic} group then {@link Optional#absent()} will be
+     * returned.
+     *
+     * @param statisticGroup The type of statistic to return
+     * @param entityType The entity type for the statistic to return
+     * @return The entity statistic or Optional.absent() if not found
+     */
+    Optional<EntityStatistic> getEntityStatistic(StatisticGroup statisticGroup, EntityType entityType);
```

Comment:
Not a fan of this to be honest, the EntityType is not  really a 'holder' of the statistic but rather just an object referenced by the group.